.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+*.sh
```

Comment:
new line at the end of the file

Reply:
merged before i saw this. I'll get it in the next ticket